src/main/kotlin/org/oso/AppInitialData.kt

```diff
@@ -39,20 +39,17 @@ class AppInitialData
         deviceTypeRepository.save(typeApp)
 
         val flicButton = Device(
-            name = "Flic Button",
-            description = "",
             deviceType = typeFlic
         )
+        flicButton.id = "Flic Button"
```

Comment:
We don´t need to handle a flic-button device server side in a special way. 
For our server system it´s just a smartphone device. 
The client app is handling the whole flic button interaction, there is no difference between a signal for example sended by a pressed flic button or a pressed GUI button.

Reply:
@pwilken Indeed. Good point!